libs/verifiable-credentials/build.gradle

```diff
@@ -16,8 +16,7 @@ dependencies {
 			"software.amazon.awssdk:dynamodb-enhanced:$rootProject.ext.dependencyVersions.dynamodbEnhanced",
 			"software.amazon.lambda:powertools-logging:$rootProject.ext.dependencyVersions.powertoolsLogging",
 			"software.amazon.lambda:powertools-parameters:$rootProject.ext.dependencyVersions.powertoolsParameters",
-			project(":libs:common-services"),
-			project(":libs:gpg45-evaluator")
```

Comment:
Notice this dependency had to be removed for the refactor to work. This meant 3 classes had to return to `common-services` because both `cimit-services` and `verifiable-credentials` depend on those classes